### DISCUSSION TITLE
Testikantojen uusinta 12.-13.12.

### DISCUSSION THREAD

**Discussion by CodoDerDritte** [KohaSuomi/Koha#969] [Huolto- ja häiriötiedotteet]
Testikantojen uusiminen 12.-13.12. voi aiheuttaa tietokantapalvelinten välillä replikointiviivettä, joten SQL-raportit eivät välttämättä ole ihan ajantasaisia työn ollessa käynnissä. Tiedotamme tässä keskusteluketjussa mahdollisista viiveistä. Testikannat tietysti ovat työn ajan myös pois käytöstä.

**Comment by paxed** [comment]
Vaara-test redusoitu viime yönä, mutta vaatii vielä ainakin anonymisoinnin ja api-käyttäjien palauttamisen.

**Reply by AnneliO** [reply]
Koitin kirjautua vaara-testille, mutta se pyytää kaksivaiheisen kirjautumisen koodia ja jos annan Vaaran tuotannon koodin, tulee syötön jälkeen Virhe 500.

**Reply by AnneliO** [reply]
Pasi (paxed) poistaa kaksivaiheisen kirjautumisen käytöstä.

**Comment by paxed** [comment]
Helle-test redusoitu.

**Comment by paxed** [comment]
Lumme-test redusoitu.

**Comment by paxed** [comment]
kyyti-test, outi-test, ja lappi-test redusoitu.

**Comment by paxed** [comment]
Ylläolevat kannat myös anonymisoitu.

**Comment by johannaraisa** [comment]
Yllämainitut kannat on nyt indeksoitu uudelleen.

**Comment by paxed** [comment]
kirkes-test redusoitu ja anonymisoitu.

**Reply by johannaraisa** [reply]
Nyt on tämäkin indeksoitu uudestaan.

**Comment by paxed** [comment]
siili-test redusoitu ja anonymisoitu.

**Reply by johannaraisa** [reply]
Tämäkin on nyt indeksoitu uudestaan.
